content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: SLSA level 2 compliance for Eclipse Temurin
+date: "2022-11-11T12:00:00+00:00"
+author: sxa
+description: Eclipse Temurin by Adoptium is compliant with level 2 of the SLSA secure development framework.
+tags:
+  - Temurin
+  - Security
+---
+
+# Introduction to SLSA
+
+[SLSA](https://slsa.dev) is a framework with individual levels to indicate
+increasing levels of security which software products can work towards. 
+Today, we are pleased to announce that the Eclipse Temurin build and
+distribution process is compliant with Level 2 of the SLSA specification.
+
+What does this mean in practice? The [SLSA v0.1 levels are defined at this
+link](http://slsa.dev/spec/v0.1/levels) and this means we are compliant with
+levels 1 and 2 of the specification.
+
+## SLSA Level 1
+
+Level 1 means that the build process is full scripted and automated and we
```

Comment:
The first sentence is a run-on sentence. Recommend splitting it into at least two sentences.